### PR TITLE
Api mapping

### PR DESCRIPTION
**Pull request by oleksiidn**
Mapped rest of payment API. Update list of building block

### CODE REVIEW

File: spec/happy-flow.md [hunk 1]
```diff
@@ -86,11 +83,60 @@ Check if the payment due date is reached, trigger the benefit payment to the cit
...
+      "PayeeFunctionalID": "string",//5
+      "Amount": 0, //6
+      "Currency": "string", // 7
+      "Narration": "string" // 8
+    }
+  ]
+}
+```
+1. Globally unique ID which is coming from [driver app](happy-flow.md#implementation).
+2. ID which is coming from [Information Mediator](https://github.com/GovStackWorkingGroup/sandbox-bb-information-mediator) to identify the origination of the request.
```
Comment (VladislavEmilTodorov) [inline spec/happy-flow.md]: Maybe this is not IM (it should be transient). I guess it should be the BB that is calling that API endpoint. In our case looks like it should be our APP (driver/mocked SRIS)

File: spec/happy-flow.md [hunk 2]
```diff
@@ -86,11 +83,60 @@ Check if the payment due date is reached, trigger the benefit payment to the cit
...
+      "PayeeFunctionalID": "string", //5
+      "Amount": 0, //6
+      "Currency": "string", //7
+      "Narration": "string" //8
+    }
+  ]
+}
+```
+1. Globally unique ID which is coming from [driver app](happy-flow.md#implementation).
+2. ID which is coming from [Information Mediator](https://github.com/GovStackWorkingGroup/sandbox-bb-information-mediator) to identify the origination of the request.
```
Comment (VladislavEmilTodorov) [inline spec/happy-flow.md]: Same as https://github.com/GovStackWorkingGroup/sandbox/pull/4/files#r1197547987

Comment (VladislavEmilTodorov) [reply spec/happy-flow.md]: I guess same for Beneficiary onboarding APIs

**Review by VladislavEmilTodorov — CHANGES_REQUESTED** [review]
Source BB ID should be defined, is it the IM id or it is the actual BB that calls that API endpoint

File: spec/happy-flow.md (hunk 1 above)
Comment (oleksiidn) [reply spec/happy-flow.md]: d778970cbcfb92e98d0a04535cf2b676033da0f7

File: spec/happy-flow.md (hunk 2 above)
Comment (oleksiidn) [reply spec/happy-flow.md]: d778970cbcfb92e98d0a04535cf2b676033da0f7

**Review by VladislavEmilTodorov — APPROVED** [review]

**Pull request merged**